### ISSUE TITLE
Looks like 22.10 and 22.04.1 tags got removed?

### ISSUE THREAD

**Issue by chenrui333** [strasdat/Sophus#486]
👋 while checking the latest release for the project, I found the two tags got referenced in the homebrew side do not exist in the repo anymore, did they get removed? Thanks!!

relates to:
- Homebrew/homebrew-core#99714
- Homebrew/homebrew-core#112860

**Comment by strasdat** [comment]
@chenrui333 Yes, apologies. Please use https://github.com/strasdat/Sophus/tree/1.22.10 and https://github.com/strasdat/Sophus/tree/1.22.4 instead.

I'm planning to release sophus2 this year, and got really unhappy with the versioning scheme I introduced in 2022. Hence, I had to map a few versions and corresponding releases accordingly:

22.04 -> 1.22.4
22.10 -> 1.22.10

@chenrui333 please let me know if / how I can help.

**Comment by strasdat** [comment]
@chenrui333, I hope this is the correct fix: https://github.com/Homebrew/homebrew-core/pull/125258

I'm new to homebrew, fingers crossed.
